docs/Tutorials/Basics/Conversations.md

```diff
@@ -0,0 +1,245 @@
+    setup steps!
+    
+    * You don't need any experience in creating quests. This is the very beginning.
+
+## 1. Package Setup
+
+//TODO: Replace these instructions with a package downloader insturction
+Let's start with a bit of preparation creating a folder structure for this tutorial. 
+All files related to quests must be placed inside the :material-folder-home: *QuestPackages* directory, which is
+automatically created by the plugin.
+
+Please create folders and files so your file structure looks like this:
+
+* :material-folder-home: QuestPackages *(already there)*
+    - :material-folder-open: tutorialQuest
+        - :material-file: package.yml
+          - :material-folder-open: conversations
+              - :material-file: jack.yml  
+    
+
+This means that the _QuestPackages_ folder must contain a folder called _tutorialQuest_. This folder contains a file named _package.yml_
+and another directory called _conversations_. The _conversations_ directory must contain another file named _jack.yml_.
+
+## 2. Linking a conversation to an NPC
+
+Usually, conversations happen between an NPC and the player. 
+Therefore, we need to create the `npcs` section in the _package.yml_ so that the plugin knows which Citizens NPC
+uses which conversation. This is how it works:
+
+``` YAML title="package.yml" linenums="1"
+npcs:
+  '1': Jack
+```
+This links the NPC with the given ID (`1`) to the conversation with the given identifier (`Jack`).
+Save the file after editing.
+
+??? info "How to find the Citizens NPC ID"
+    ??? info "How to create a NPC" 
+        Execute this command: `/npc create Jack`
+                
+    1. Stay close to the NPC you want to select
+    2. Type the command `/npc select` to select the nearest NPC.
+    3. Type the command `/npc id` to get the ID from your NPC.
+
+
+## 3. Creating your first conversation
+
+It's time to create the first conversation with Jack! This chapter will teach you the basic structure of a conversation.
+
+A conversation always starts with an `NPC_option`, which is text the NPC says to the player. Now the player must
+answer the NPC using a `player_option` which is also some text. Usually, a player has more than one answer to choose from.
+These are determined by the `pointer` argument, that points from the initial `NPC_option` to all possible `player_options`.
+After the player responded, they are shown another `NPC_option` that the previously chosen `player_option` points to.
+  
+As you can see, a conversation mainly consists of three elements: `NPC_options`, `player_options` and 
+the `pointer` argument. 
+
+Whenever either a `player_option` or a `NPC_option` point to no other option the conversation ends as there are no more
+responses or answers.
+
+!!! info "The Conversation Cycle"
+    ``` mermaid
+    graph LR
+    X{Conversation Starts} --> C
+    C[Initial NPC_option] --> A
```

Comment:
```suggestion
    C[First NPC_option] --> A
```
I think we should go with the same name as it is called in the files here. Also `first` is easier language.